### PR TITLE
README: add Homebrew installation instructions

### PR DESCRIPTION
**Pull request by waldyrious**
...and other small tweaks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,16 +64,17 @@ cargo install --path .
...
 
-diffr tries to be a well behaved unix program: it reads its input from stdin
+diffr tries to be a well behaved Unix program: it reads its input from stdin
 and writes to stdout.
 
-#### To test it:
+#### One-off usage
+
 ```
 cargo build --release
```
Comment (waldyrious) [inline README.md:73]: Should this line be here, @mookid? How does it interact with the the section above about building from source?

Comment (mookid) [reply README.md:73]: It was meant to be closer to the top of the readme, and predates the installation instructions. You can indeed remove that section.

**Comment by mookid** [comment]
thanks!

File: README.md (hunk 1 above)
Comment (waldyrious) [reply README.md:73]: > You can indeed remove that section.

Not sure I follow. It seems strange that you'd suggest removing the entire [Installation section](https://github.com/mookid/diffr#installation). Did you perhaps mean remove the `cargo build --release` line?

Comment (mookid) [reply README.md:73]: yes, I am talking about thse (now misplaced) lines.

Comment (waldyrious) [reply README.md:73]: Ok, I've submitted #54 to remove that line.

**Pull request merged**